types/tar/index.d.ts

```diff
@@ -249,7 +249,7 @@ export interface FileStat extends stream.Readable, Fields {
     size: number;
 }
 
-export interface ReadEntry extends MiniPass, HeaderProperties {
+export interface ReadEntry extends MiniPass.default, HeaderProperties {
```

Comment:
minipass v3.3.5 types use `export =` instead of having a default export, so you should just be able to extend `MiniPass`:

```suggestion
export interface ReadEntry extends MiniPass, HeaderProperties {
```

Reply:
The tests seem to pass fine through the PR CI, so you might've accidentally installed `minipass` in the root project's `node_modules` on your local setup. Run `npm install` inside the types folder instead to avoid the error